app/src/main/java/com/android/shelfLife/ui/newoverview/NewOverview.kt

```diff
@@ -41,26 +38,14 @@ import kotlinx.coroutines.launch
  * Composable function to display the overview screen
  *
  * @param navigationActions The actions to handle navigation
- * @param houseHoldRepository The repository to handle household data
  */
 @Composable
-fun OverviewScreen(
-    navigationActions: NavigationActions,
-    houseHoldRepository: HouseHoldRepository,
-    listFoodItemRepository: FoodItemRepository,
-    userRepository: UserRepositoryFirestore
-) {
-  val overviewScreenViewModel = viewModel {
-    OverviewScreenViewModel(
-        houseHoldRepository = houseHoldRepository,
-        listFoodItemsRepository = listFoodItemRepository,
-        userRepository = userRepository)
-  }
+fun OverviewScreen(navigationActions: NavigationActions) {
+  val overviewScreenViewModel = hiltViewModel<OverviewScreenViewModel>()
 
   val selectedHousehold by overviewScreenViewModel.selectedHousehold.collectAsState()
   val foodItems by overviewScreenViewModel.foodItems.collectAsState()
   val households by overviewScreenViewModel.households.collectAsState()
-  val householdViewModelIsLoaded by overviewScreenViewModel.finishedLoading.collectAsState()
```

Comment:
If I understand correctly this is unecessary because we already have can check if it has been loaded or not by this statement `selectedHousehold == null && households.isEmpty()`